build.gradle

```diff
@@ -23,6 +23,10 @@ intellij {
 dependencies {
     compile group: 'org.eclipse.lsp4j', name: 'org.eclipse.lsp4j', version: '0.6.0'
     compile group: 'com.vladsch.flexmark', name: 'flexmark', version: '0.34.58'
+    testCompile group: 'junit', name: 'junit', version: '4.11'
+    testCompile group: 'org.mockito', name: 'mockito-all', version: '1.6.5'
```

Comment:
Use version `1.10.19` as that is compatible with the version of powermock you are using.